swan-lake/learn/keeping-ballerina-up-to-date.md

```diff
@@ -12,87 +12,91 @@ redirect_from:
+Also, there is a plan to develop a native compiler called nBallerina in the future, which will target platforms such as Linux, Windows and Mac OS.
 
-### The Ballerina Tool
+Ballerina distribution is a term, which refers to these jBallerina and nBallerina compilers.
 
-**Ballerina** is a command-line tool for managing Ballerina source code. It helps you to manage Ballerina projects and modules, test, build, and run programs, etc.
+### Getting to know the Release Channels
 
-It also enables you to easily install, update, and switch among Ballerina distributions. The main focus of this guide is to teach you how to perform these actions but first, let’s talk about Ballerina distributions.
+Ballerina distributions are released via two different release channels at the moment. One is the stable 1.x release channel and the other one is the Swan Lake release channel.
 
-### Ballerina Distributions
+Ballerina yet does not have a release channel for nightly builds that give you access to the latest perhaps unstable features.
 
-- The language specification defines the syntax and semantics of Ballerina programming language. Ballerina compiler is a software program that validates the Ballerina source code and translates it to an executable program. There exist a production-ready official compiler called  jBallerina. We also have a plan to do a native compiler called nBallerina.
-- jBallerina
-  - Ballerina compiler that targets the JVM.
-  - The most stable and production-ready compiler.
-- nBallerina
-  - Ballerina compiler that targets platforms such as Linux, Windows and Mac OS.
-  - Not available yet.
+#### Versioning of the releases
 
-Ballerina distribution is a term that we use to refer to jBallerina and nBallerina compilers.
+Both the above channels of Ballerina distribution releases will strictly follow [SemVer](https://semver.org/) with `major.minor.patch` version numbers.
 
-### Release Channels
+#### Patch Releases
 
-Ballerina distributions are released on two different release channels at the moment: patch releases and minor releases. Both these channels distribute stable versions. Ballerina yet does not have a release channel for nightly builds that give you access to the latest perhaps unstable features.
-
-Ballerina distribution releases strictly follow [SemVer](https://semver.org/) with major.minor.patch version numbers.
-
-#### Patch Release Channel
-
-This channel gives you access to the patch releases of Ballerina distributions that contain bug fixes and fixes for critical stability and security related issues. These releases are strictly time-bound and happen every two weeks. Occasionally,  you would see on-demand patch releases as well.
+Patch releases of Ballerina distributions contain bug fixes and fixes for critical stability and security related issues. These releases are strictly time-bound and happen every two weeks. Occasionally,  you would see on-demand patch releases as well.
```

Comment:
`These releases are strictly time-bound and happen every two weeks.` I think we can't say this exactly

Reply:
Removed.